[DOC-2231] feat(RBAC): add docs for query ownership privilege; (#487)
tigergraph/server-docs: modules/user-access/pages/fine-grained-query-privileges.adoc

* `WRITE_QUERY` privilege is deprecated. It has been split into privileges such as `CREATE_QUERY`, `UPDATE_QUERY` and `DROP_QUERY`.

* New privileges `INSTALL_QUERY`, `EXECUTE_QUERY` and `OWNERSHIP` are added.

[NOTE]
====

> `OWNERSHIP` is a special privilege introduced since 4.1.0. It should be granted to the user created this query automatically. 

Reviewer: ```suggestion
`OWNERSHIP` is a special privilege introduced in version 4.1.0.
It is granted automatically tp the user who creates a query.
```

The entity (a user or a role) who owns a query holds all privileges on a query. That means, this entity can read, update, drop, install and execute this query.

> At the same time, this entity can grant the query privileges to other entities, including the `OWNERSHIP` privilege. Once the `OWNERSHIP` privilege is granted to another entity, the previous owner should lose `OWNERSHIP` privilege on this query.

Reviewer: ```suggestion
An entity with `OWNERSHIP` privilege (e.g., the creator) on a query can grant privileges to other entities, including the `OWNERSHIP` privilege. Once the `OWNERSHIP` privilege is granted to another entity, the previous owner loses `OWNERSHIP` privilege on this query.
[NOTE]
====
Each object can have only one explicit owner entity.  
====
```

Reviewer: @jue-yuan: "Should" is the wrong word.
"Should" means "it is the right or correct way".  It also implies that the wrong or incorrect way could happen, so it introduces uncertainty.  We do not want uncertainty about whether TigerGraph implements the language rules or not.

Users with the `superuser` role should hold the `OWNERSHIP` privilege on all queries in all graphs implicitly.

> Users with the `admin` role in one graph should hold the `OWNERSHIP` privilege on all queries in this graph implicitly.

Reviewer: ```suggestion
Users with the `superuser` role hold the `OWNERSHIP` privilege on all queries in all graphs implicitly.

Users with the `admin` role in one graph hold the `OWNERSHIP` privilege on all queries in this graph implicitly.
```

====

* Query privilege scope changes:
** `CREATE_QUERY` privilege can only be granted/revoked at global and graph level.
** `READ_QUERY`, `UPDATE_QUERY`, `DROP_QUERY`, `INSTALL_QUERY` and `EXECUTE_QUERY` privileges can only be granted/revoked at individual query level.

> ** `OWNERSHIP` privilege can only be granted at individual query level and only be granted to one entity, e.g.: a user or a user-defined role.

Reviewer: ```suggestion
** `OWNERSHIP` privilege can only be granted at the individual query level and only to one entity, e.g.: a user or a user-defined role.
```

        READ_QUERY
        UPDATE_QUERY
----

> .Ex. Grant `OWNERSHIP` privilege on existing queries q3 in graph `g1` to user `u1`:

Reviewer: ```suggestion
.Ex. Grant `OWNERSHIP` privilege on existing query q3 in graph `g1` to user `u1`:
```
